packages/cli/generators/project/templates/package.plain.json

```diff
@@ -14,35 +14,35 @@
     "build": "tsc --outDir dist --target es2017",
     "build:watch": "tsc --outDir dist --target es2017 --watch",
     "clean": "rm -rf dist",
-    <% if (project.prettier && project.tslint) { -%>
+<% if (project.prettier && project.tslint) { -%>
```

Comment:
No need to change the formatting for this file. Please revert.

If you have a build script to do auto-formatting, please exclude template files.

Reply:
@raymondfeng With the previous formatting, the indenting on the generated package.json was inconsistent. The indenting is consistent with the new format.